frontend/public/components/utils/resource-link.tsx

```diff
@@ -121,6 +127,13 @@ export const ResourceLink: React.FC<ResourceLinkProps> = ({
           {nameSuffix}
         </span>
       )}
+      {isArgoCDResource(metadata) && (
```

Comment:
We'll want to generalize this so we can handle any managed-by label, not just Argo CD.